src/routes/createexercise/schema.ts

```diff
@@ -13,7 +13,11 @@ export const formSchema = z.object({
 		.string()
 		.min(1, 'You must create a solution')
```

Comment:
Er der en grund til, at der er et max og et min på code solution?

Reply:
Ikke rigtigt, kan godt være man kunne have bruge for mere?